app/pages/search/SearchPage.js

```diff
@@ -74,7 +74,6 @@ class UnconnectedSearchPage extends Component {
           showMap={showMap}
         />
         <div className="app-SearchPage__content">
-          <h1>{t('SearchPage.title')}</h1>
```

Comment:
It's still needed in the page title (in DocumentTitle component). But yeah, otherwise it would be best to remove it as well.